spec/mongoid/interceptable_spec.rb

```diff
@@ -170,6 +170,65 @@ class TestClass
     it "runs after document instantiation" do
       expect(game.name).to eq("Testing")
     end
+
+    context 'when the document is embedded' do
+
+      before do
+        Book.destroy_all
```

Comment:
Why is this needed? AFAIK MongoDB is [purged](https://github.com/mongodb/mongoid/blob/095433fa53fc5f770fedb8d28b57fa33b2b2c4dc/spec/spec_helper.rb#L124-L127) before any test.